docs/DeveloperGuide.md

```diff
@@ -51,22 +58,24 @@ For example, the `Logic` component (see the class diagram given below) defines i
 
 **How the architecture components interact with each other**
```

Comment:
This statement should reference the picture above. Currently the picture `LogicClassDiagram.png` is not referenced in the DG

Reply:
Okay done. Thanks!